boilerplate/flyte/end2end/run-tests.py

```diff
@@ -243,6 +248,12 @@ def run(
     is_flag=False,
     help="Name of domain in project to run functional tests on",
 )
+@click.option(
+    "--parallel",
+    is_flag=True,
+    default=False,
```

Comment:
Run flytesnacks workflow serially in the CI due to disk pressure in the sandbox.